mpas_analysis/ocean/climatology_map_argo.py

```diff
@@ -0,0 +1,461 @@
+    fields
+
+    Authors
+    -------
+    Luke Van Roekel, Xylar Asay-Davis
+    """
+
+    def __init__(self, config, mpasClimatologyTask,
+                 refConfig=None):  # {{{
+        """
+        Construct the analysis task.
+
+        Parameters
+        ----------
+        config :  ``MpasAnalysisConfigParser``
+            Configuration options
+
+        mpasClimatologyTask : ``MpasClimatologyTask``
+            The task that produced the climatology to be remapped and plotted
+
+        refConfig :  ``MpasAnalysisConfigParser``, optional
+            Configuration options for a reference run (if any)
+
+        Authors
+        -------
+        Luke Van Roekel, Xylar Asay-Davis
+        """
+        fieldName = 'temperatureARGO'
+        # call the constructor from the base class (AnalysisTask)
+        super(ClimatologyMapArgoTemperature, self).__init__(
+                config=config, taskName='climatologyMapArgoTemperature',
+                componentName='ocean',
+                tags=['climatology', 'horizontalMap', 'argo', fieldName])
+
+        sectionName = self.taskName
+
+        mpasFieldName = 'timeMonthly_avg_activeTracers_temperature'
+        iselValues = None
+
+        # read in what seasons we want to plot
+        seasons = config.getExpression(sectionName, 'seasons')
+
+        if len(seasons) == 0:
+            raise ValueError('config section {} does not contain valid list '
+                             'of seasons'.format(sectionName))
+
+        comparisonGridNames = config.getExpression(sectionName,
+                                                   'comparisonGrids')
+
+        if len(comparisonGridNames) == 0:
+            raise ValueError('config section {} does not contain valid list '
+                             'of comparison grids'.format(sectionName))
+
+        depths = config.getExpression(sectionName, 'depths')
+
+        if len(depths) == 0:
+            raise ValueError('config section {} does not contain valid '
+                             'list of depths'.format(sectionName))
+
+        # the variable 'timeMonthly_avg_landIceFreshwaterFlux' will be added to
+        # mpasClimatologyTask along with the seasons.
+        remapClimatologySubtask = RemapDepthSlicesSubtask(
+            mpasClimatologyTask=mpasClimatologyTask,
+            parentTask=self,
+            climatologyName=fieldName,
+            variableList=[mpasFieldName],
+            seasons=seasons,
+            depths=depths,
+            comparisonGridNames=comparisonGridNames,
+            iselValues=iselValues)
+
+        if refConfig is None:
+
+            refTitleLabel = 'Roemmich-Gilson Argo Climatology: Temperature'
+
+            observationsDirectory = build_config_full_path(
+                config, 'oceanObservations', 'argoSubdirectory')
+
+            obsFileName = \
+                '{}/ArgoClimatology_TS.nc'.format(
+                        observationsDirectory)
+            refFieldName = 'theta'
```

Comment:
Could we rename this to 'thetaArgo' so the corresponding mapping file is clearly for ARGO?  So far, there's no conflict but I could imagine one in the future.

Reply:
done